finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -10,24 +10,22 @@
  *     IBM Corporation - Initial implementation
  *******************************************************************************/
 // end::copyright[]
+// tag::manager[]
 package io.openliberty.guides.inventory;
 
 import java.util.Properties;
-import io.openliberty.guides.inventory.client.SystemClient;
-import io.openliberty.guides.inventory.model.InventoryList;
 import javax.enterprise.context.ApplicationScoped;
+import io.openliberty.guides.inventory.model.InventoryList;
 
-// tag::ApplicationScoped[]
 @ApplicationScoped
-// end::ApplicationScoped[]
 public class InventoryManager {
 
   private InventoryList invList = new InventoryList();
-  private SystemClient systemClient = new SystemClient();
+  private InventoryUtils invUtils = new InventoryUtils();
+
+  public Properties get(String hostname, int portNumber) {
+    Properties properties = invUtils.getProperties(hostname, portNumber);
```

Comment:
how come you need a portNumber here? I dont recall it being necessary in the other guides?

Reply:
The config guide uses it, and the health guide build on top of the config guide, so it's there in the first place.

Reply:
ah ok